docker/docker-compose.yaml

```diff
@@ -12,6 +12,22 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data/
 
+  grafana:
+    image: "grafana/grafana:latest"
+    environment:
+      GF_DATABASE_TYPE: postgres
+      GF_DATABASE_HOST: db:5432
+      GF_DATABASE_NAME: jobserver
```

Comment:
this is being used to store grafana's own data - should we have created another service e.g. `db-grafana` and use db_name `grafana` here?

Reply:
(For anyone reading this in future) Yep, you're absolutely right, I should have done that or let it default to a sqlite db! The env vars to configure the postgres datasource are very similar to this and I think I copied the wrong ones.